nrrd/reader.py

```diff
@@ -453,7 +454,7 @@ def read_data(header, fh=None, filename=None, index_order='F'):
     fh.close()
 
     if total_data_points != data.size:
-        raise NRRDError('Size of the data does not equal the product of all the dimensions: {0}-{1}={2}'
+        raise NRRDError('Size of the data does not equal the product of all the dimensions: {}-{}={}'
```

Comment:
Might as well go all the way and just use a f-string since Python 3.7 is required. Can you fix any other occurrences of `.format` if any?

Reply:
yeah, this should be caught by the `pyupgrade` hook. It has been working really well for things like this in other projects, I will fix this manually and search for more `format` ocurrences